<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backgroup:rgb(232, 235, 237);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>371</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(165, 35, 28);
border-radius:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_n1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(200, 212, 229);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(255, 255, 255);
background-color: rgb(128, 137, 210);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Número 1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_n2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(200, 212, 229);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_n3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(200, 212, 229);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_n4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(200, 212, 229);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_n5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(200, 212, 229);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(255, 255, 255);
background-color: rgb(128, 137, 210);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Número 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(255, 255, 255);
background-color: rgb(128, 137, 210);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Número 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(255, 255, 255);
background-color: rgb(128, 137, 210);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Número 4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(255, 255, 255);
background-color: rgb(128, 137, 210);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Número 5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_calculador">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(44, 63, 112);
background-color: rgb(232, 235, 237);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>     Calculador de promedio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calcularpromedio">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 63, 112);

font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color:rgb(255, 255, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string> Promedio </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
